config/locales/views/criteria/en.yml

```diff
@@ -8,6 +8,7 @@ en:
     description_hint: You may use Markdown in this field. Save changes to see a preview.
     editing_criteria: Editing criteria
     errors:
+      different_assignment: 'There exists criteria under different assignment:'
```

Comment:
This message should be displayed when the criterion id is for a criterion of a different assignment, *or* an id that doesn't exist in the database at all. That means the wording should be a bit different, something like "Could not find criteria for this assignment with the given ids". With this in mind, please also pick a better name for the key for this string (instead of `different_assignment`).

Also, don't end the message with a colon (the ids don't automatically display). Using no punctuation is okay, this is consistent with how we display other flash error messages.